lib/lita/adapters/slack/rtm_connection.rb

```diff
@@ -64,7 +64,7 @@ def send_messages(channel, strings)
         end
 
         def shut_down
```

Comment:
Is this method user anywhere outside of here because inside of this class it is no longer used (not sure if that was intentional